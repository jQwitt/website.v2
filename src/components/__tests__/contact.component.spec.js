import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Contact from '../contact.component';

afterEach(cleanup);
describe('Contact renders correctly', () => {
  test('snapshot', () => {
    const { asFragment } = render(<Contact />);
    expect(asFragment()).toMatchSnapshot();
  });
});
