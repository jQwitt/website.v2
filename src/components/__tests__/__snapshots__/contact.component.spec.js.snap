// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact renders correctly snapshot 1`] = `
<DocumentFragment>
  <div
    class="col-12 text-center contact"
  >
    <div>
      <h1>
        Lets Talk!
      </h1>
      <div
        class="social-media-links d-flex justify-content-center"
      >
        <a
          href="https://www.linkedin.com/in/joseph-witt-52a960171/"
        >
          <i
            class="flaticon-linkedin large"
          />
        </a>
        <a
          href="mailto:jackwittbiz@gmail.com"
        >
          <i
            class="flaticon-email large"
          />
        </a>
        <a
          href="https://github.com/jQwitt"
        >
          <i
            class="flaticon-browser large"
          />
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
